components/calendar/src/cal/hijri.rs

```diff
@@ -58,9 +57,16 @@ fn year_as_hijri(standard_era: tinystr::TinyStr16, year: i32) -> types::YearInfo
 ///
 /// This calendar is a pure lunar calendar with no leap months. It uses month codes
 /// `"M01" - "M12"`.
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug)]
 pub struct HijriObservational {
-    data: Option<DataPayload<CalendarHijriObservationalV1>>,
+    pub(crate) location: HijriObservationalLocation,
+    data: Option<DataPayload<ErasedMarker<HijriCache<'static>>>>,
+}
+
+#[derive(Clone, Debug, Copy, PartialEq)]
+pub enum HijriObservationalLocation {
```

Comment:
This definitely should be non-exhaustive

Reply:
+1

Reply:
it's actually private, I should update the visibility